src/main/resources/banner.txt

```diff
@@ -0,0 +1,9 @@
+ __        ___    _     _     _____ _____           ____  _____ ______     _____ ____ _____
+ \ \      / / \  | |   | |   | ____|_   _|         / ___|| ____|  _ \ \   / /_ _/ ___| ____|
+  \ \ /\ / / _ \ | |   | |   |  _|   | |    _____  \___ \|  _| | |_) \ \ / / | | |   |  _|
+   \ V  V / ___ \| |___| |___| |___  | |   |_____|  ___) | |___|  _ < \ V /  | | |___| |___
+    \_/\_/_/   \_\_____|_____|_____| |_|           |____/|_____|_| \_\ \_/  |___\____|_____|
```

Comment:
[performance]

```suggestion
__        ___    _     _     _____ _____     _______     _______ _   _ _____ 
\ \      / / \  | |   | |   | ____|_   _|   | ____\ \   / / ____| \ | |_   _|
 \ \ /\ / / _ \ | |   | |   |  _|   | |_____|  _|  \ \ / /|  _| |  \| | | |  
  \ V  V / ___ \| |___| |___| |___  | |_____| |___  \ V / | |___| |\  | | |  
   \_/\_/_/ __\_\_____|_____|_____|_|_|___ _|_____|__\_/__|_____|_| \_| |_|  
      |  _ \_ _/ ___||  _ \ / \|_   _/ ___| | | | ____|  _ \                 
 _____| | | | |\___ \| |_) / _ \ | || |   | |_| |  _| | |_) |                
|_____| |_| | | ___) |  __/ ___ \| || |___|  _  | |___|  _ <                 
      |____/___|____/|_| /_/   \_\_| \____|_| |_|_____|_| \_\                
```

Reply:
Btw, if you want to explore variations I used: https://www.asciiart.eu/text-to-ascii-art